CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How to contribute to Project Curve2Code
+
+First off, Thank you for your interest in contributing to this project. Contributions form Person(s) like you are what makes this project better and more effective. Without further ado, here are the ways you can contribute to this project.
+
+1. Opening issues.
+2. Submitting changes.
+
+## Getting Started
+
+* Make sure you have a [Github account](https://github.com/signup/free).
+* Make sure you have read the [Code of conduct](https://github.com/svijaykoushik/curve2code/blob/master/CODE_OF_CONDUCT.md).
+* Get familiar with [creating issues on Github](https://help.github.com/articles/creating-an-issue/),
+* Get familiar with [creating pull requests on Github](https://help.github.com/articles/creating-a-pull-request/).
+
+## Opening Issues
+
+You can create an issue when
+
+1. Something is not working as described in the project.
+2. Something is not working as you expected it to work.
+3. Suggesting an idea that can improve the project.
+
+### Choosing an issue template
+
+Curve2Code provides two types of templates on the project's [issue page](https://github.com/svijaykoushik/curve2code/issues) to choose from
+
+1. Template for bug reporting and
+2. Template for suggesting enhancements and new ideas.
+
+You can choose the template which suits your issue(s) or choose to submit a regular issue template if your issue(s) don't fit in these two template.
+
+### Guidelines to open an issue
+
+Irrespective of the type of issue, all issues should follow these guidelines
+
+* Ensure that your issue is not already listed on the issue tracker page.
+* Provide a short and descriptive title for the issue.
+* Make a clear and concise description of the issue.
+* Make a clear and concise description of the impact of the issue on the project.
+
+## Submitting changes
+
+You can submit bug fixes and enhancements to the project by creating a "Pull request".
```

Comment:
Change to: You can submit bug fixes and enhancements to the project by
* cloning the project
* Make your change(s).
* Propose the change(s) creating a "Pull Request"